examples/interface-add.yaml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I suggest to drop the example manifests if you also have them in the README file.
This way you don't have to maintain two copies of the same thing.

Reply:
I find it easy to use manifests directly from the text using the shell's [here doc](https://en.wikipedia.org/wiki/Here_document#:~:text=In%20computing%2C%20a%20here%20document,it%20were%20a%20separate%20file.) feature:
```bash
cat <<EOF | kubectl apply -f -
---
apiVersion: "k8s.cni.cncf.io/v1"
kind: NetworkAttachmentDefinition
metadata:
  name: macvlan1-config
spec:
  config: '{
            "cniVersion": "0.4.0",
            "plugins": [
                {
                    "type": "macvlan",
                    "capabilities": { "ips": true },
                    "master": "eth1",
                    "mode": "bridge",
                    "ipam": {
                        "type": "static"
                    }
                }, {
                    "type": "tuning"
                } ]
        }'
EOF
```

Reply:
Yes, but how will any automation consume that from the readme ?

Reply:
maybe doing but using the example link would be a compromise ...

That would not throw the yaml at the readme users face, but the reader will know what to look for.